### PR TITLE
Level screens

### PR DESCRIPTION
**Pull request by adriennedomingus**
Mostly just want to be able to see the diff more clearly at this point, not looking for a merge.

### CODE REVIEW

**Comment by erinnachen** [comment]
👍

**Comment by adriennedomingus** [comment]
@erinnachen Ready for a code review. Changes on the branch:

Draw top and side borders: they are just borders, not "floors"
Draw level layouts
Add a level attribute to dino that increments once all windups have been caught and turned into collected fruit
Current level fades out, new level fades in
Canvas size increased: jump size and distance a bubble travels before turning up increased proportionally. Tests adjusted accordingly.

File: test/dinosaur-test.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ describe('dinosaur', function(){
   it('hits the floor by default', function(){
     this.dino.y++;
     var result = this.dino.onAFloor(this.floors);
+    console.log(result);
```
Comment (erinnachen) [inline test/dinosaur-test.js]: this should come out.

File: lib/game_play.js [hunk 2]
```diff
@@ -84,3 +85,30 @@ export function drawFloors(floors, context){
     floor.draw(context);
   });
 }
+
+export function levelUp(dino, fruits, windup, canvas) {
+  if (!windup && allFruitsCollected(fruits) && fruits.length > -1 + dino.level ) {
```
Comment (erinnachen) [inline lib/game_play.js]: is this conditional to check that the level is over?

File: test/dinosaur-test.js (hunk 1 above)
Comment (adriennedomingus) [inline test/dinosaur-test.js]: Done.

File: lib/game_play.js (hunk 2 above)
Comment (adriennedomingus) [inline lib/game_play.js]: Yep. I'll pull it out with that name.

**Comment by erinnachen** [comment]
@adriennedomingus one last thing... did we make sure that the fruit doesn't fall through the floor?

**Comment by erinnachen** [comment]
🎉🐬🎉 merging.

**Pull request merged**